guides/source/form_helpers.md

```diff
@@ -405,96 +404,109 @@ Here you have a list of cities whose names are presented to the user. Internally
 The most generic helper is `select_tag`, which - as the name implies - simply generates the `SELECT` tag that encapsulates an options string:
 
 ```erb
-<%= select_tag(:city_id, '<option value="1">Lisbon</option>...') %>
+<%= select_tag(:city_id, raw('<option value="1">Lisbon</option><option value="2">Madrid</option><option value="3">Berlin</option>')) %>
 ```
 
 This is a start, but it doesn't dynamically create the option tags. You can generate option tags with the `options_for_select` helper:
 
 ```html+erb
-<%= options_for_select([['Lisbon', 1], ['Madrid', 2], ...]) %>
+<%= options_for_select([['Lisbon', 1], ['Madrid', 2], ['Berlin', 3]]) %>
+```
 
-output:
+Output:
 
+```html
 <option value="1">Lisbon</option>
 <option value="2">Madrid</option>
-...
+<option value="3">Berlin</option>
 ```
 
-The first argument to `options_for_select` is a nested array where each element has two elements: option text (city name) and option value (city id). The option value is what will be submitted to your controller. Often this will be the id of a corresponding database object but this does not have to be the case.
-
-Knowing this, you can combine `select_tag` and `options_for_select` to achieve the desired, complete markup:
```

Comment:
The flow here was better before with this phrase. The guides should not be mechanical but have a nice flow